ethanschreur/kids-krafts
# Customers can view available products on the shop page

**What does this PR do?**
- This pull request allows customers to view products that are available for sale on the "shop" page.

**Description of the task to be completed** 
- If products in the database have the category "selling", display them on the "shop" page/

**How should this  be tested**
- (On Windows) Run `. venv/scripts/activate` 
- `python run.py`
- Go to http://127.0.0.1:5000/login
- Enter the correct email and password and submit them
- View the dashboard page
- Press the products button
- Fill out the "add products" form and submit it
- Click on the added product in the products table
- Fill out the "add subproducts" form and submit it
- Add some more products with subproducts, some with the category "selling" and others with the category "not selling".
- Go to http://127.0.0.1:5000/shop
- Make sure the products you added that have the category "selling" are displayed, while the ones with the category "not selling" are not displayed.
